src/user/using-the-aggregate-data-entry-app.md

```diff
@@ -0,0 +1,283 @@
+# Using the Data Entry app (beta) { #aggregate_data_entry_app } 
+
+## About the Data Entry app { #aggregate_data_entry_app.about } 
+
+The **Data Entry** app (beta) is where you manually enter aggregated data in
+DHIS2. You register data for an organisation unit, a period, and a set
+of data elements (data set) at a time. A data set often corresponds to a
+paper-based data collection tool. You configure the data sets in the
+**Maintenance** app.
+
+> **Note**
+> 
+> If a data set has both a section form and a custom form, the system
+> displays the custom form during data entry. Users who enter data can't
+> select which form they want to use. In web-based data entry the order
+> of display preference is:
+> 
+> 1.  Custom form (if it exists)
+> 
+> 2.  Section form (if it exists)
+> 
+> 3.  Default form
+> 
+> Mobile devices do not support custom forms. In mobile-based data entry
+> the order of display preference is:
+> 
+> 1.  Section form (if it exists)
+> 
+> 2.  Default form
+
+When you close an organisation unit, you can't register or edit data to
+this organisation unit in the **Data Entry** app (beta).
+
+## Selecting a form { #aggregate_data_entry_app.selecting_a_form }
+
+![](resources/images/data_entry_beta/selecting_a_form.png)
+
+In order to show a form, the following information is required:
+
+1.  Open the **Data Entry** app (beta).
+
+1.  A data set
+
+    As selectable the organisation unit and period depend on the data set, the
+    data set has to be chosen first.
+
+1.  An organisation unit
+
+1.  A period
+    
+    The available periods are controlled by the period type of the data
+    set (reporting frequency). You can jump a year back or forward by
+    clicking **Prev year** or **Next year**.
+
+1.  Attribute options (in case the data set has an attribute option combo)
+
+    The attribute option combo also depends on the period and organisation
+    unit, so the input will be hidden until all required values have been
+    selected. If a data set does not have an attribute option combo, then no
+    selector will appear.
```

Comment:
I'm not sure that we should talk about "attribute option combo's" in end user documentation - that's quite a technical description. 
I think we need to talk simply about "options"?

```suggestion
1.  Additional options

    The available options also depend on the period and organisation
    unit, so the input will be hidden until all required values have been
    selected. If a data set does not have an additional options, then no
    selector will appear.
```

Reply:
The app also supports option sets, which are category options for a particular field (I think..). The top bar also has a dropdown button with the label "Options", so that word can have multiple meanings.

If you think it's still fine to simply refer to "options" when talking about selecting a form, I can commit the suggestion

Reply:
Perhaps we can run this by @cooper-joe ?

The point is that I think end users shouldn't have to care what is an attribute option, or a category option, or whatever; all they need to know is that there are additional categories* that they have to select.

*from a purely end-user point of view I would probably prefer to refer to "additional categories" than "additional options"; so one modification of my suggestion would be to change `options` to `categories` throughout that paragraph.

Reply:
Could we refer to them as `Additional selections`? `Selections` is ambiguous regarding what they actually are (options, category options, etc.) but rather refers to the action of selecting. 

I also think the wording here is a quite technical, I've attempted to make a pass at making it slightly more user friendly:
```
Additional selections (dependent on previous choices)
If there are any other selections available, they’ll be shown as the last controls in the top bar. Additional selections depend on the chosen data set, organisation unit, and period, so they won’t be shown until those first three selections are made. If there aren’t any additional selections, the control won’t be shown.
```